### PR TITLE
PD-2329 update stable and prod configs for awardee trans_am (tach).

### PR DESCRIPTION
**Pull request by Seeker1911**
update stable and prod config to whitelist ip ranges for awardee-tach.

### CODE REVIEW

File: rest-api/config/config_prod.json [hunk 1]
```diff
@@ -17,8 +17,14 @@
...
     "exporter@all-of-us-rdr-prod.iam.gserviceaccount.com": {
       "roles": ["exporter"]
-    }
-  },
+    },
+    "awardee-tach@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
+      "roles": ["awardee_sa"],
+      "awardee": "TRANS_AM",
+      "whitelisted_ip_ranges" : {
+        "ip4": ["66.189.127/27", "68.124.42.162/27"]
```
Comment (danqrodney) [inline rest-api/config/config_prod.json]: You don't need a fourth part of 66.189.127?

File: rest-api/config/config_stable.json [hunk 2]
```diff
@@ -36,6 +36,13 @@
         "whitelisted_ip_ranges" : {
           "ip4": ["10.50.178.54"]
         }
+    },
+      "awardee-tach@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
+        "roles": ["awardee_sa"],
+        "awardee": "TRANS_AM",
+        "whitelisted_ip_ranges" : {
+          "ip4": ["66.189.127/27", "68.124.42.162/27"]
```
Comment (danqrodney) [inline rest-api/config/config_stable.json]: See above.

File: rest-api/config/config_prod.json (hunk 1 above)
Comment (Seeker1911) [reply rest-api/config/config_prod.json]: No, it will cover .0-.32. It's redundant.

File: rest-api/config/config_stable.json (hunk 2 above)
Comment (Seeker1911) [reply rest-api/config/config_stable.json]: done

**Comment by Seeker1911** [comment]
Added description field.

**Pull request merged**